src/adapters/catalonia.js

```diff
@@ -107,4 +107,95 @@ const getLatestMeasurements = function (measurements) {
     }
   });
   return Object.values(latestMeasurements);
-}
\ No newline at end of file
+}
+// remove stations deemed to be duplicates of EEA stations
+function filterDuplicates(measurements, criteria) {
+  return measurements.filter(measurement => {
+    return !criteria.some(criterion => {
+      const matchLocation = measurement.location === criterion.location;
+      const matchCity = measurement.city === criterion.city;
+      const matchCoordinates =
+        measurement.coordinates.latitude === criterion.coordinates.latitude &&
+        measurement.coordinates.longitude === criterion.coordinates.longitude;
+      const matchParameter = !criterion.parameter || measurement.parameter === criterion.parameter;
+
+      return matchLocation && matchCity && matchCoordinates && matchParameter;
+    });
+  });
+}
+
+const blacklist = [
+  {
+      "location": "Alcover",
+      "city": "Alcover",
```

Comment:
It looks like the location and the city fields are always the same so a bit redundant we could just remove one